std/json.d

```diff
@@ -1719,3 +1727,37 @@ pure nothrow @safe unittest // issue 15884
     Test!dchar();
 }
 
+@safe unittest // issue 15885
+{
+    static bool test(const double num0)
+    {
+        import std.math : feqrel;
+
+        const json0 = JSONValue(num0);
+        const num1 = to!double(toJSON(json0));
+        const json1 = JSONValue(num1);
+        const num2 = to!double(toJSON(json1));
+        import std.stdio;
+        writefln("%.18g : %.18g : %.18g : %s", num0, num1, num2, feqrel(num2, num0));
+
+        version(Windows)
```

Comment:
> What 's happening on Windows ?

I don't know. `Win_32` failed on `win-farm-2`at `assert(test(1.223e+24))`, so I added the debug printout and re-ran the test... which then passed on `win-farm-1`.

Originally I thought it was just imprecision in the Windows `snprintf()` - which is indirectly called by `toJSON()` - but now I'm wondering if it's actually a code gen bug, or maybe flaky hardware on one/both of the Windows test servers.